scripts/apps/monitoring/directives/MonitoringGroup.ts

```diff
@@ -310,6 +310,10 @@ export function MonitoringGroup(cards, api, authoringWorkspace, $timeout, superd
             scope.refreshGroup = function() {
                 monitoring.showRefresh = scope.showRefresh = false;
                 scheduleQuery(null, {force: true});
+                // update scroll position to top when forced refresh
+                if (containerElem[0].scrollTop > 0) {
```

Comment:
Why is the conditional used instead of always updating it?

Reply:
to avoid scroll event which is triggered even if you set it from 0 to 0, we trigger refresh when user scrolls to top.